### PR TITLE
Feature/extemding wrapper for omop

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**
Could we merge this part now? That way we can start using the OMOP base image for creating algorithms. Dont delete the branch.

### CODE REVIEW

File: docker/algorithm-base.Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
-FROM python:3.10-slim-buster
+FROM python:3.11-slim-bullseye
```
Comment (bartvanb) [inline docker/algorithm-base.Dockerfile:1]: If we do this, we need to modify the docs that algorithms should be written in python 3.11.
Also, what's the diff between bullseye and buster?

File: docker/algorithm-base.Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 
+# install dependencies for the ohdsi tools part of the wrapper
```
Comment (bartvanb) [inline docker/algorithm-base.Dockerfile:3]: This comment seems not to belong?

File: docker/algorithm-omop-base.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+RUN Rscript -e "install.packages('DatabaseConnector')"
+RUN Rscript -e "install.packages('drat')"
+RUN Rscript -e "drat::addRepo('OHDSI'); install.packages('FeatureExtraction')"
+RUN Rscript -e "install.packages('remotes')"
+RUN Rscript -e "remotes::install_github('ohdsi/CirceR')"
+RUN Rscript -e "remotes::install_github('OHDSI/CohortDiagnostics')"
+RUN Rscript -e "remotes::install_github('OHDSI/CohortGenerator')"
+
+
+ENV LD_LIBRARY_PATH=/usr/lib/jvm/java-17-openjdk-amd64/lib/server/
```
Comment (bartvanb) [inline docker/algorithm-omop-base.Dockerfile:33]: Maybe you could add a comment what this does

**Review by bartvanb — CHANGES_REQUESTED** [review]

File: docker/algorithm-base.Dockerfile (hunk 1 above)
Comment (frankcorneliusmartin) [reply docker/algorithm-base.Dockerfile:1]: That is the version of the Debian distribution. I needed this for some R / Java related dependency. This OS version change should not affect algorithms. 

I will check if it also works with python:3.10, else I will update the docs.

File: docker/algorithm-omop-base.Dockerfile (hunk 3 above)
Comment (frankcorneliusmartin) [reply docker/algorithm-omop-base.Dockerfile:35]: I added a comment and a fixme, as this is not such a great solution. However i also do not know how with deal with this properly.

File: docker/algorithm-base.Dockerfile (hunk 2 above)
Comment (frankcorneliusmartin) [reply docker/algorithm-base.Dockerfile:3]: Yes you ar right.

File: docker/algorithm-base.Dockerfile (hunk 1 above)
Comment (frankcorneliusmartin) [reply docker/algorithm-base.Dockerfile]: I ended up reverting to `3.10` just to avoid compatibility issues

Comment (bartvanb) [reply docker/algorithm-base.Dockerfile]: Clear!

**Review by bartvanb — APPROVED** [review]

**Pull request merged**